content/swift/concepts/sets/terms/remove.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '.remove()'
+Description: 'Removes an element at a specified index.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.remove()`** method removes an element at a specified index.
+
+## Syntax
+
+```pseudo
+setName.remove(at: index)
+```
+
+## Example
+
+```swift
+var weather: Set = ["spring", "summer", "autumn", "fall", "winter"]
+let toRemove = "autumn"
+if let _ = weather.remove(toRemove) {
+  print(weather)
+}
+```
+
+In the example above, The `"autumn"` element in the `weather` array was removed. This will output:
```

Comment:
```suggestion
This will output the following:
```

Reply:
Thank you I fixed this !

Reply:
Hmmm looks like the fix didn't go through?

Reply:
I thought you pushed it. I'm sorry. I just posted it Please check